packages/vue/src/composables/useSelect.ts

```diff
@@ -0,0 +1,7 @@
+import { ref } from 'vue';
+
+// state
+
+const active = ref(false);
+
```

Comment:
This maybe overkill, but I think it will be used later when vue adds real primitives. For now it's a simple reactive value that helps keep track of what tab is added.